docs/smart-contracts/modules.md

```diff
@@ -1,40 +1,38 @@
 ---
-title: Modules and references
-description: "Modules that define the business logic and essential functions for blockchain applications and provide the basic foundation for all Pact smart contracts."
+title: Modules and interfaces
+description: "Modules define the business logic and essential functions for blockchain applications and provide the basic foundation for all Pact smart contracts."
 id: modules
 sidebar_position: 7
 ---
 
-# Modules and references
+# Modules and interfaces
 
-The fundamental building blocks for all Pact smart contracts are defined as Pact **modules**.
+The fundamental building blocks for all Pact smart contracts are defined in Pact **modules**.
```

Comment:
The section is named `Modules` and `interfaces` but does not describe what `interfaces ` are.